src/app/shared/forge.enhance.service.ts

```diff
@@ -61,7 +61,9 @@ export class EnhancedForgeService extends ForgeService {
           if (input.valueChoices) {
             input.valueChoices.forEach(choice => {
               let key = input.name + (choice.key ? choice.key : choice.id);
-              choice.description = this.asciidoc.generateHtml(key);
+              if (this.asciidoc.generateHtml(key)) {
```

Comment:
How costly is calling this function twice?

Reply:
yes, good one let's improve that